types/steam-user/index.d.ts

```diff
@@ -0,0 +1,5456 @@
+    chats: Record<string, Chat>;
+
+    /**
+     * An object whose keys are 64-bit SteamIDs, and whose values are values from the `EFriendRelationship` enum.
+     * When we get unfriended, instead of setting the value to `EFriendRelationship.None`, the key is deleted from the object entirely.
+     * This isn't populated after logon until `friendsList` is emitted.
+     */
+    myFriends: Record<string, SteamUser.EFriendRelationship>;
+
+    /**
+     * An object whose keys are 64-bit SteamIDs, and whose values are from the `EClanRelationship` enum.
+     * When we leave a group, instead of setting the value to `EClanRelationship.None`, the key is deleted from the object entirely.
+     * This isn't populated after logon until `groupList` is emitted.
+     */
+    myGroups: Record<string, SteamUser.EClanRelationship>;
+
+    /**
+     * An object containing your friend groups (in the official client, these are called tags). Keys are numeric group IDs.
+     */
+    myFriendGroups: Record<string, { name: string; members: SteamID[] }>;
+
+    /**
+     * An object containing the nicknames you have assigned to other users.
+     * Keys are numeric 64-bit SteamIDs, properties are strings containing that user's nickname.
+     * This is empty until `nicknameList` is emitted.
+     */
+    myNicknames: Record<string, string>;
+
+    /**
+     * An object containing cached data about known apps and packages. Only useful if the `enablePicsCache` option is `true`.
+     */
+    picsCache: { changenumber: number; apps: Record<string, any>; packages: Record<string, any> };
+
+    /**
+     * Contains the name of this package. This allows other modules to verify interoperability.
+     */
+    packageName: 'steam-user';
+
+    /**
+     * Contains the version of this package. For example, "4.2.0". This allows other modules to verify interoperability.
+     */
+    packageVersion: string;
+
+    // EVENTS
+    on<K extends keyof Events>(event: K, listener: (...args: Events[K]) => void): this;
+    once<K extends keyof Events>(event: K, listener: (...args: Events[K]) => void): this;
+    off<K extends keyof Events>(event: K, listener: (...args: Events[K]) => void): this;
+    removeListener<K extends keyof Events>(event: K, listener: (...args: Events[K]) => void): this;
+    removeAllListeners(event?: keyof Events): this;
+
+    /**
+     * Set a configuration option.
+     * @param option
+     * @param value
+     */
+    setOption(option: keyof Options, value: any): void;
+
+    /**
+     * Set one or more configuration options
+     * @param options
+     */
+    setOptions(options: Options): void;
+
+    /**
+     * Set a sentry file
+     * @param sentry Binary Sentry File
+     */
+    setSentry(sentry: Buffer | null): void;
+
+    // logOn
+    logOn(details?: LogOnDetailsAnon | LogOnDetailsNamePass | LogOnDetailsNameKey | LogOnDetailsNameToken): void;
+
+    /**
+     * Log off of Steam gracefully.
+     */
+    logOff(): void;
+
+    // relog
```

Comment:
```suggestion
```
please remove comments like this. Better to have those empty, otherwise please use a proper comment:
```ts
/**
 * Log off of Steam gracefully.
 */
 ```

Reply:
fair point 😀